.github/workflows/kind-e2e.yaml

```diff
@@ -30,6 +30,10 @@ jobs:
       with:
         path: ./src/github.com/${{ github.repository }}
 
+    - name: Build ko images
```

Comment:
You probably already saw that the E2E failed. You might want to enable `GO111MODULE: off` in the workflow file to fix this. Also this step must come after you set up `kind` because the image injection would not work.

Reply:
The `ko` build step is still failing, IMHO bc you still use GOPATH and not mod. Try with mod and it should work.

Reply:
okay, setting `GO111MODULE: on` right before the ko command seems to work. Where should I have turned it off then, like you mention in the first comment?